src/main/java/picard/sam/markduplicates/ElcHashBasedDuplicatesFinder.java

```diff
@@ -40,13 +40,20 @@
  */
 class ElcHashBasedDuplicatesFinder extends ElcDuplicatesFinder {
 
-    // We split each read on several parts and each part has length hashLength.
-    // hashLength = (min read length) / (number of errors + 1)
-    private int hashLength = Integer.MAX_VALUE;
+    // We split each read on several parts, each part represent set of nucleotides,
```

Comment:
I would word it as follows: "We split the bases of each read into interspersed parts, so that each part spans the entire length of the read. e.g. breaking a read of length 12 into 4 parts would be as follows:"